src/TypeRegistry.php

```diff
@@ -81,51 +81,52 @@ public static function init() {
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/EditLock.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/MediaItemStatusEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/MediaItemSizeEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/MediaDetails.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/MediaItemMeta.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/MediaSize.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/MenuItem.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Input/MenuItemsConnectionWhereArgs.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/MenuLocationEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Menu.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/MimeTypeEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/OrderEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/PageInfo.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Plugin.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Input/PostObjectsConnectionOrderbyInput.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/PostObjectsConnectionOrderbyEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/PostObjectsConnectionDateColumnEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/PostObjectFieldFormatEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/PostStatusEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/PostType.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/PostTypeLabelDetails.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/RootMutation.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/RootQuery.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/PostTypeEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/RelationEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Settings.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/TermObjectsConnectionOrderbyEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/TimezoneEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Theme.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Taxonomy.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/TaxonomyEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/User.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/UsersConnectionSearchColumnEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/UserRole.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/UserRoleEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/SettingGroup.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/PostObject.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/TermObject.php';
+		require_once( WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Avatar.php' );
```

Comment:
Any reason for adding the brackets here? Seems like this is what's causing CI to fail

Reply:
not sure where they came from - I've recently started working on a macbook and I assume my IDE thought they were needed for PHP.